workspace/manifest-writer/src/index.ts

```diff
@@ -11,7 +11,7 @@ import { sortKeysByPriority } from '@pnpm/object.key-sorting'
 async function writeManifestFile (dir: string, manifest: Partial<WorkspaceManifest>): Promise<void> {
   manifest = sortKeysByPriority({
     priority: { packages: 0 },
-    deep: false,
+    deep: true,
```

Comment:
I am not sure about this. For instance, overrides might be better not sorted.

Reply:
I personally think that sorting doesn't matter. Of course, if you insist, maybe we should extend the `sortKeysByPriority` function.

Reply:
I don't have objections but we have to verify that the order in "overrides" doesn't matter. It might make a difference in what order we search for the override.

Reply:
I am not sure sorting of keys should be excluded for overrides. I just said we need to check if the order matters.

Reply:
I simply checked it using vite and esbuild and found that the order had no effect on the results.